<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>378</width>
    <height>409</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>15</pointsize>
   </font>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow {
	background-color:transparent;
}
QDialog {
	background-color: rgb(54, 54, 54);
	border-radius: 12px;
	
}




</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QToolButton" name="btn_window_close">
       <property name="styleSheet">
        <string notr="true">QToolButton {
    width:30px;
    height:30px;
    border-radius: 12px;
    line-height:40px;
    border:1px solid #4F95DA;;
    color:#4F95DA;
    text-align:center;
    text-decoration:none;
    font-size:20px;
    font-family:oswald;
}

QToolButton:hover {
	color:#fff;
    top:0;
    left:0;
    width:100%;
    height:100%;
    background:#4F95DA;
    border:1px solid transparent;
}
</string>
       </property>
       <property name="text">
        <string>X</string>
       </property>
       <property name="iconSize">
        <size>
         <width>60</width>
         <height>60</height>
        </size>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <spacer name="verticalSpacer_3">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>17</width>
         <height>37</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="styleSheet">
        <string notr="true">QLabel {
    width:166px;
    height:38px; 
    line-height:40px;  
    color:#4F95DA;
    text-align:center;
    text-decoration:none;
    font-size:28px;
    font-family:oswald;
}
</string>
       </property>
       <property name="text">
        <string>AUTO LOGIN</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lbl_last_user">
       <property name="styleSheet">
        <string notr="true">QLabel {
    color:#4F95DA;
    text-align:center;
    text-decoration:none;
    font-size:18px;
    font-family:oswald;
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>17</width>
         <height>37</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="btn_confirm">
       <property name="sizeIncrement">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
    width:66px;
    height:38px;
    border-radius: 12px;
    line-height:40px;
    border:1px solid #4F95DA;;
    color:#4F95DA;
    text-align:center;
    text-decoration:none;
    text-transform:uppercase;
    font-size:14px;
    font-family:oswald;
}

QPushButton:hover {
	color:#fff;
    top:0;
    left:0;
    width:100%;
    height:100%;
    background:#4F95DA;
    border:1px solid transparent;
}
</string>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::No|QDialogButtonBox::Yes</set>
       </property>
       <property name="centerButtons">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>17</width>
         <height>37</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="main.qrc"/>
 </resources>
 <connections/>
 <designerdata>
  <property name="gridDeltaX">
   <number>10</number>
  </property>
  <property name="gridDeltaY">
   <number>10</number>
  </property>
  <property name="gridSnapX">
   <bool>true</bool>
  </property>
  <property name="gridSnapY">
   <bool>true</bool>
  </property>
  <property name="gridVisible">
   <bool>true</bool>
  </property>
 </designerdata>
</ui>
